### PR TITLE
Fixed broken links in Markdown Contributing and Readme files

### PR DESCRIPTION
**Pull request by rodyvansambeek**
## Why make this change?

- Fixes broken links in Markdown Contributing and Readme files

## What is this change?

- Fix broken links

### CODE REVIEW

**Comment by rodyvansambeek** [comment]
@microsoft-github-policy-service agree

**Review by severussundar — APPROVED** [review]
LGTM! Thanks for contributing this change.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Before running the code, ensure you have the correct version of .NET installed (
 
 The next step is to ensure you have a config file for DAB defined. If you want to use the sample one [you can use the generator tool](#an-alternative-way-to-generate-config-files), otherwise, ensure that the `dab-config.<engine>.json` file is in the `src/Service` directory (and if the database is CosmosDb_NoSql the GraphQL schema file too).
 
-Make sure the config has a valid connection string, or you use the [`@env` syntax to reference an environment variable](docs/configuration-file.md#accessing-environment-variables).
+Make sure the config has a valid connection string, or you use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/en-us/azure/data-api-builder/configuration-file#accessing-environment-variables).
```
Comment (seantleonard) [inline CONTRIBUTING.md:27]:
```suggestion
Make sure the config has a valid connection string or use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/azure/data-api-builder/configuration-file#accessing-environment-variables).
```

removing `en-us` will allow browser to determine locale for non-US based readers.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ To run the SQL tests locally you will need to:
 
 1. Setup a database using the server(s) that you want to test against.
 1. Create the database schema using the `DatabaseSchema-<engine>.sql` file.
-1. Set the connection string in `dab-config.<engine>.json` or use the [`@env` syntax to reference an environment variable](docs/configuration-file.md#accessing-environment-variables).
+1. Set the connection string in `dab-config.<engine>.json` or use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/en-us/azure/data-api-builder/configuration-file#accessing-environment-variables).
```
Comment (seantleonard) [inline CONTRIBUTING.md:57]:
```suggestion
1. Set the connection string in `dab-config.<engine>.json` or use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/azure/data-api-builder/configuration-file#accessing-environment-variables).
```

removing `en-us` will allow browser to determine locale for non-US based readers.

File: samples/azure/readme.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Azure Deploy Sample
 
-Use the provided `azure-deploy.sh` script to deploy Data API builder in an Azure Container Instance as described in [Running Data API Builder in Azure](/docs/running-in-azure.md)
+Use the provided `azure-deploy.sh` script to deploy Data API builder in an Azure Container Instance as described in [Running Data API Builder in Azure](https://learn.microsoft.com/en-us/azure/data-api-builder/running-in-azure)
```
Comment (seantleonard) [inline samples/azure/readme.md:3]:
```suggestion
Use the provided `azure-deploy.sh` script to deploy Data API builder in an Azure Container Instance as described in [Running Data API Builder in Azure](https://learn.microsoft.com/azure/data-api-builder/running-in-azure)
```

File: samples/azure/readme.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
 
-Use the provided `azure-container-apps-deploy.sh` script to deploy Data API builder in an Azure Container Apps as described in [Running Data API Builder in Azure](/docs/running-in-azure.md)
\ No newline at end of file
+Use the provided `azure-container-apps-deploy.sh` script to deploy Data API builder in an Azure Container Apps as described in [Running Data API Builder in Azure](https://learn.microsoft.com/en-us/azure/data-api-builder/running-in-azure)
```
Comment (seantleonard) [inline samples/azure/readme.md:5]:
```suggestion
Use the provided `azure-container-apps-deploy.sh` script to deploy Data API builder in an Azure Container Apps as described in [Running Data API Builder in Azure](https://learn.microsoft.com/azure/data-api-builder/running-in-azure)
```

**Review by seantleonard — CHANGES_REQUESTED** [review]
Thank you for contributing! Links to Learn.Microsoft.com should exclude locale (e.g. `en-us), since the browser will auto-resolve that value.

File: CONTRIBUTING.md (hunk 1 above)
Comment (rodyvansambeek) [reply CONTRIBUTING.md]: Thanks, excuse my oversight. Changed the URL without the locale.

**Comment by rodyvansambeek** [comment]
> Thank you for contributing! Links to Learn.Microsoft.com should exclude locale (e.g. `en-us), since the browser will auto-resolve that value.

Thanks, excuse my oversight. I have changed the URL's without the locale.

**Review by seantleonard — APPROVED** [review]
Thank you for authoring these updates and addressing comments :)

**Pull request merged**